contents/docs/libraries/node/index.mdx

```diff
@@ -77,7 +77,7 @@ A `capture` call requires:
 Optionally, you can also submit:
 
 -   `properties`, which is an object with any information you'd like to add
--   `sendFeatureFlags`, a boolean that determines whether to send current known feature flags with this event. This is useful when running experiments which depends on this event. However, when this is set to `true`, we will make an additional request to fetch the current feature flags and thus slow things down. A workaround is to [manually compute feature flag information](/tutorials/experiments#step-2-sending-the-right-events) so that we can avoid making the request.
+-   `sendFeatureFlags`, a boolean that determines whether to send current known feature flags with this event. This is useful when running experiments which depends on this event. However, when this is set to `true`, we will make an additional request to fetch the current feature flags and thus slow things down. A workaround is to [manually compute feature flag information](/tutorials/experiments#step-2-sending-the-right-events ???) so that we can avoid making the request.
```

Comment:
Im not sure what the right update here is. I dont knoow what "manually compute the feature flag information" means, and I feel that it wasn't pointing to the right link in the first place?

This also appears in the python and ruby docs (you'll see my changes below highlighting them with "???"

Reply:
Ah yeah I think this means adding `$feature/` properties yourself. The way you do that is by using local evaluation to get all properties, then format them in the same way as step 2, then add them to the event.